CefSharp/AcceleratedPaintInfo.cs

```diff
@@ -2,12 +2,17 @@
 //
 // Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
 
+using System;
+using CefSharp.Enums;
+
 namespace CefSharp
 {
     /// <summary>
     /// AcceleratedPaintInfo
     /// </summary>
     public sealed class AcceleratedPaintInfo
     {
+        public IntPtr SharedTextureHandle;
```

Comment:
These should be immutable properties

Please add a `get` and set via constructor. for both properties

Also please include the appropriate xml doc based on the `CEF API`.